Src/Private/Get-AbrFgtSystem.ps1

```diff
@@ -319,11 +481,89 @@ function Get-AbrFgtSystem {
                 }
             }
 
+            # Fetch HA Configuration
+            $haConfig = Get-FGTSystemHA
+            $haPeers = Get-FGTMonitorSystemHAPeer
+            $haChecksums = Get-FGTMonitorSystemHAChecksum
+
+            if( $haConfig.mode -ne 'standalone' -and $infoLevel.System -ge 1) {
+                Section -Style Heading3 'High Availability' {
+                    Paragraph "The following section details HA settings."
+                    BlankLine
+                
+                    Section -Style Heading4 'HA Configuration' {
+                        $OutObj = @()
+    
+                        $OutObj = [pscustomobject]@{
+                            "Group Name / ID / Mode" = $haConfig.'group-name'+' / '+$haConfig.'group-id'+' / '+$haConfig.mode
+                            "HB Device" = $haConfig.'hbdev'
```

Comment:
can you align (indent) '=' ? (see visual code configuration for auto formatting)